reactjs-jsx-ppx/reactjs_jsx_ppx_v3.ml

```diff
@@ -1315,5 +1312,5 @@ let jsxMapper =
 
 let () =
   Driver.V2.register_transformation "reactjs"
-    ~impl:(fun ctxt str -> jsxMapper#structure ctxt str)
-    ~intf:(fun ctxt sig_ -> jsxMapper#signature ctxt sig_)
+    ~preprocess_impl:(fun ctxt str -> jsxMapper#structure ctxt str)
+    ~preprocess_intf:(fun ctxt sig_ -> jsxMapper#signature ctxt sig_)
```

Comment:
Why does react ppx need to change to `preprocess_*` arguments? Won't this play badly with other ppxs?

Reply:
indeed. this needs to be reverted. I was just trying things out.